### ISSUE TITLE
Code pour l'appli "Livradois-Forez rando" ?

### ISSUE THREAD

**Issue by dolmen** [GeotrekCE/Geotrek-mobile#299]
Est-ce la base de code qui a servi à construire l'appli "Livradois-Forez Rando" sur Android?

Je pose la question car je vois un copyright Makina-Corpus sur les deux.

Et aussi parce que je constate que les liens entre l'appli et le site https://www.livradois-forez-rando.fr/ (où je vois mentionné Geotrek) ne marchent plus (404).

**Comment by camillemonchicourt** [comment]
Oui, l'outil Geotrek-mobile est bien l'outil opensource qui est utilisé pour générer l'application mobile "Livradois-Forez Rando", ainsi que de nombreuses autres applications mobiles de randonnées de différents territoires.

La dernière en date étant "Alès Agglo Evasion" (https://play.google.com/store/apps/details?id=io.ales.geotrek&hl=fr&gl=US).

Pour "Livradois-Forez Rando", il faudrait voir directement avec le Parc naturel régional Livradois-Forez si ils ont modifié quelque chose de leur côté, ou si ils ne maintiennent plus l'application mobile, au profil du site web et mobile actuellement disponible sur https://www.livradois-forez-rando.fr.

@relaire, une idée ?

**Comment by bastyen** [comment]
> Et aussi parce que je constate que les liens entre l'appli et le site https://www.livradois-forez-rando.fr/ (où je vois mentionné Geotrek) ne marchent plus (404).

Quels liens ?

**Comment by dolmen** [comment]
@bastyen 
Depuis le descriptif d'une rando dans l'appli Android :
* le téléchargement de PDF
* le téléchargement de GPX

**Comment by bastyen** [comment]
Je pense que les urls ne sont plus valables suite à la mise en place du Geotrek Rando v3. De même les url de partage ne doivent plus l'être.
Il faudrait mettre à jour l'application mobile ou créer des redirections.
